app/home/components/mainSection/index.jsx

```diff
@@ -20,50 +22,12 @@ const MainSection = ({ email, subscribeEmail, handleEmailChange, shown }) => (
             value={email}
             type="text"
           />
-          <button className={styles.emailSubmitButton}>Subscribe</button>
+          <button className={styles.emailSubmitButton}>
+            <Icon icon="RIGHT_SIDE_ARROW" />
+          </button>
         </form>
       </div>
-
-      <div className={styles.followSection}>
-        <a
-          onClick={() => trackAndOpenLink("https://www.facebook.com/Pluto-263226227503100/", "Footer")}
-          className={styles.footerIconWrapper}
-        >
-          <Icon icon="FACEBOOK_FOOTER" />
-        </a>
-        <a
-          onClick={() => trackAndOpenLink("https://medium.com/pluto-network", "MainSection")}
-          className={styles.footerIconWrapper}
-        >
-          <Icon icon="MEDIUM_FOOTER" />
-        </a>
-        <a
-          onClick={() => trackAndOpenLink("https://twitter.com/pluto_network", "Footer")}
-          className={styles.footerIconWrapper}
-        >
-          <Icon icon="TWITTER_FOOTER" />
-        </a>
-        <a onClick={() => trackAndOpenLink("https://t.me/plutonetwork", "Footer")} className={styles.footerIconWrapper}>
-          <Icon icon="TELEGRAM_FOOTER" />
-        </a>
-        <a
-          onClick={() => trackAndOpenLink("https://github.com/pluto-net", "Footer")}
-          className={styles.footerIconWrapper}
-        >
-          <Icon icon="GITHUB_FOOTER" />
-        </a>
-      </div>
-
-      <div className={styles.backgroundImage}>
-        <img src="https://d2vo77dayzjoat.cloudfront.net/researchers.png" alt="" />
-      </div>
-    </div>
-
-    <div className={styles.waveIcon}>
-      <Icon icon="WAVES" />
-    </div>
-    <div className={styles.bubblesIcon}>
-      <Icon icon="BUBBLES" />
+      <Icon icon="SCROLL_INDECATOR" />
```

Comment:
Doesn't it do any function like scroll down?

Reply:
It just an indicator. but I think when it clicked, scrolling down to next section could be cool... Can you help me...?